hazelcast/src/main/java/com/hazelcast/client/impl/statistics/Statistics.java

```diff
@@ -52,18 +57,18 @@
 
     private final MetricsRegistry metricsRegistry;
     private final boolean enabled;
-    private final HazelcastProperties properties;
     private final ILogger logger = Logger.getLogger(this.getClass());
 
     private final HazelcastClientInstanceImpl client;
 
     private final boolean enterprise;
+    private final ClientMetricsConfig metricsConfig;
 
     private PeriodicStatistics periodicStats;
 
     public Statistics(final HazelcastClientInstanceImpl clientInstance) {
```

Comment:
`Statistics` and `ClientStatistics` - this classes (and objects) are really different, but their names are almost identical. I know that it was that way before this PR, but maybe changing `Statistics` to something like `ClientStatisticsService` (or any more meaningful name) will help to improve readability?

Reply:
Agreed. Renamed to `ClientStatisticsService`.